tests/metadata/test_parse_metadata.py

```diff
@@ -84,9 +84,13 @@ def test_from_file(self):
         assert "number_string" in metadata.labels
         assert metadata.labels["number_string"] == "1234abcde"
         assert "123-432" in metadata.labels
-        assert metadata.owners == ["test1@mozilla.com", "test2@example.com"]
-        assert metadata.labels["owner1"] == "test1"
-        assert metadata.labels["owner2"] == "test2"
+        assert metadata.owners == [
+            "test1@mozilla.com",
+            "test2@example.com",
+            "mozilla/test_group",
+        ]
+        assert metadata.labels["owner1"] == "test1@mozilla.com"
+        assert metadata.labels["owner2"] == "test2@example.com"
```

Comment:
Could we add a test to assert the case of owner "mozilla/test_group"? Expects an exception (or None).